File: 58-LengthOfLastWord/58-LengthOfLastWord.js
// Last updated: 3/7/2026, 10:02:53 PM
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    let count = 0;
    for(let i=s.length-1; i>=0; i--){
        if(s[i] !== ' '){
            count++;
        }
        if(s[i] === ' ' && count>0){
            return count;
        }
    }
    return count;
};

File: 58-LengthOfLastWord/58-LengthOfLastWord.cpp
// Last updated: 8/18/2026, 11:54:58 AM
class Solution {
public:
    int lengthOfLastWord(string s) {
        int count = 0;

        for(int i=s.size()-1; i>=0; i--){
            if(s[i] != ' '){
                count++;
            }
            if(s[i] == ' ' && count > 0){
                break;
            }
        }
        return count;
    }
};